packages/fxa-auth-server/docs/swagger/shared/descriptions.ts

```diff
@@ -112,8 +145,29 @@ const DESCRIPTIONS = {
   queryKeys:
     'Indicates whether a new `keyFetchToken` is required, default to `false`.',
   payload: 'Opaque payload to be forwarded to the device.',
+  paymentMethodId: 'A unique identifier for the payment method.',
+  paymentProvider:
+    'The third-party that assists in the electronic payments (e.g. PayPal, Stripe).',
+  paymentType:
+    'The type of the payment method (e.g., `credit`, `debit`, `prepaid`, or `unknown`).',
+  paypalPaymentError: 'The payment error from PayPal encountered.',
+  planId:
+    'A unique identifier for the [plan](https://stripe.com/docs/api/plans/object).',
+  planMetadata:
+    'Set of key-value pairs used to store additional information about the plan.',
```

Comment:
Perhaps we could link to the Ecosystem docs where we discuss plan metadata? Or at least metadata generally.